Assets/Scripts/Scenes/InGame/Enemy/Trace/Feature/FeatureView.cs

```diff
@@ -13,17 +16,51 @@ public class FeatureView : MonoBehaviour
         private GameObject[] _stageInteracts;
         private GameObject[] nearStageObject = null;
         private GameObject interactTarget = null;
+        private GameObject _floor;
         AudioSource _audioSource;
         private Subject<Unit> _onDestroy = new Subject<Unit>();
         public IObservable<Unit> onDestroy { get => _onDestroy; }
+        public ReactiveProperty<GameObject> TemperatureTrace = new ReactiveProperty<GameObject>();
```

Comment:
```suggestion
        public ReactiveProperty<GameObject> floor = new ReactiveProperty<GameObject>();
```
TemperatureTrace という変数名ですが、中身がGameObjectであり、RadiationFeatureがこの変数を参照しているのに違和感があります。適切な名前に変えてほしいです。